lib/libshare/nfs.c

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
I don't see any "license for new files" guidelines, but as this doesn't seem to be copied from another file (lacking copyright, at least), may be you could use "new" CDDL license text without dead links as found here: https://github.com/illumos/illumos-gate/blob/master/usr/src/prototypes/prototype.c.

Reply:
This entire file was copied from `lib/libshare/os/linux/nfs.c` (and `lib/libshare/os/freebsd/nfs.c`, but the bits that were copied out of there were originally copied from `lib/libshare/os/linux/nfs.c`, too, despite the file being marked as 2BSD instead of CDDL), so this constitutes Covered Code as seen in the third paragraph, I'm pretty sure.